### PR TITLE
Tests: Update cypress and use the newly introduced Cypress.platform

### PR DESCRIPTION
**Pull request by youknowriad**
Cypress released the `Cypress.platform` variable which allows us to switch keyboard shortcuts if necessary.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -38,14 +35,12 @@
         "onetime": {
           "version": "1.1.0",
           "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
-          "integrity": "sha1-ofeDj4MUxRbwXs78vEzP4EtO14k=",
-          "dev": true
+          "integrity": "sha1-ofeDj4MUxRbwXs78vEzP4EtO14k="
```
Comment (youknowriad) [inline package-lock.json]: I wonder why I have all these changes. I started from a clear install

Comment (gziolo) [reply package-lock.json]: You moved `cypress` from dev to regular dependencies. That might be it :)

File: test/e2e/integration/003-multi-block-selection.js [hunk 2]
```diff
@@ -37,9 +37,11 @@ describe( 'Multi-block selection', () => {
 		cy.get( lastBlockContainerSelector ).should( 'not.have.class', multiSelectedCssClass );
 
 		// Multiselect via keyboard
-		// Mac uses meta modifier so we press both here
-		cy.get( 'body' ).type( '{ctrl}a' );
-		cy.get( 'body' ).type( '{meta}a' );
+		if ( Cypress.platform === 'darwin' ) {
```
Comment (gziolo) [inline test/e2e/integration/003-multi-block-selection.js]: Can we wrap it with:

```js
 const isMacOs = Cypress.platform === 'darwin'
```
or sth similar?

File: package-lock.json (hunk 1 above)
Comment (youknowriad) [reply package-lock.json]: Good catch :) Thanks

File: test/e2e/integration/003-multi-block-selection.js (hunk 2 above)
Comment (gziolo) [reply test/e2e/integration/003-multi-block-selection.js]: Or sth like:


```js
cy.get( 'body' ).type( platformSpecific( '{meta}a', { meta: 'ctrl' } ) );
```

**Review by gziolo — APPROVED** [review]
LGTM 👍

**Pull request merged**